src/pyramid/decorator.py

```diff
@@ -36,6 +36,7 @@ class reify:
     def __init__(self, wrapped):
         self.wrapped = wrapped
         update_wrapper(self, wrapped)
+        self.__doc__ = wrapped.__doc__
```

Comment:
update_wrapper is supposed to do it :confused:

Reply:
It's pretty clear, as you said, that update_wrapper is supposed to do it and is also more defensive about doing it than this code (i.e. if `wrapped` doesn't have a doc attr). I'd want to know why the expected version isn't working first if you could step through update_wrapper and determine why it's not setting this itself.

Reply:
Maybe that's why, this would set it to `None` if not defined which would Sphinx to find it.

Reply:
According to this there's no tests for `None`, it's just pulling the value and setting it.

https://github.com/python/cpython/blob/3.9/Lib/functools.py#L50-L56